bin/release-helper.sh

```diff
@@ -195,18 +173,14 @@ function cmd-git-commit-release() {
 
     echo $1 || verify_valid_version
 
-    for file in ${VERSION_FILE} ${VERSION_PY} ${DEPENDENCY_FILE}; do
-        git add "$file" || echo "did not add file '$file' to commit"
-    done
-    git commit -m "release version ${1}"
+    git add "${DEPENDENCY_FILE}"
+    git commit --allow-empty -m "release version ${1}"
```

Comment:
Maybe add a comment here why allow empty is necessary in community even though the git add above.